.github/workflows/release.yaml

```diff
@@ -147,11 +147,16 @@ jobs:
 
       - name: Create Docker manifest
         run: |
+          echo "creating the manifest list that contains only Windows tags to preserve the os.version metadata"
+          docker manifest create ${{ env.IMAGE }}:${{ env.VERSION }} \
+              ${{ env.IMAGE }}:${{ env.VERSION }}-windows-1809-amd64 \
+              ${{ env.IMAGE }}:${{ env.VERSION }}-windows-ltsc2022-amd64 
+          docker manifest push ${{ env.IMAGE }}:${{ env.VERSION }} 
+          
+          echo "updating the manifest list to append Linux tags"
           docker buildx imagetools create --tag ${{ env.IMAGE }}:${{ env.VERSION }} \
```

Comment:
`docker buildx imagetools create` works directly against Docker Hub, while `docker manifest create` works locally and requires a push afterwards? Is that correct?

Reply:
Yes, that is correct.